ringpop.go

```diff
@@ -45,7 +45,7 @@ import (
 	"time"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/apache/thrift/lib/go/thrift"
+	athrift "github.com/apache/thrift/lib/go/thrift"
```

Comment:
Stands for Apache Thrift. To make a distinction between that and the Uber/TChannel thrift package.